.nvmrc

```diff
@@ -1 +1 @@
-v16.13.1
+v16.14.0
```

Comment:
Shall we make this v16 so it always picks latest LTS?

Reply:
(same above)

Reply:
Even better - it allows `lts` as an alias :)